### PR TITLE
APP-406: Fix Profiling and grouping demos description

### PR DESCRIPTION
**Pull request by am1kk**
http://jira.kaaproject.org/browse/APP-406

### CODE REVIEW

**Comment by am1kk** [comment]
Don't merge please

File: profilingdemo/resources/projects.xml [hunk 1]
```diff
@@ -654,22 +654,22 @@ To do this:
 		
 	</li>
 	<li>
-		Select the <b>PagerClientProfile</b> schema by clicking on the corresponding row.
+		Select the <b>Endpoint client profile schema</b> by clicking on the corresponding row.
```
Comment (hunch7) [inline profilingdemo/resources/projects.xml]: s/client-side profile schema of the endpoint
See the last bullet of this section please: http://kaaproject.github.io/kaa/docs/v0.10.0/Customization-guide/How-to-contribute/Style-guide/#grammar

File: profilingdemo/resources/projects.xml [hunk 2]
```diff
@@ -654,22 +654,22 @@ To do this:
...
 		The <b>Client-side EP profile schema details</b> page will open.
 		
 		<br />
 		<img src="https://raw.github.com/hunch7/sample-apps/app-231-EpGroup-bundle/profilingdemo/resources/pic/profiling_demo_1.png" />
 		<br />
 		<br />
 		
-		The <b>PagerClientProfile</b> schema is included in the Sandbox for demonstration purpose.
+		The <b>Endpoint client profile schema</b> is included in the Sandbox for demonstration purpose.
```
Comment (hunch7) [inline profilingdemo/resources/projects.xml]: s/client-side profile schema of the endpoint

File: profilingdemo/resources/projects.xml [hunk 3]
```diff
@@ -654,22 +654,22 @@ To do this:
...
 		You can create your own client-side EP profile schemas.
 		To do this, use the <a href="http://kaaproject.github.io/kaa/docs/v0.10.0/Programming-guide/Server-REST-APIs/#!/Profiling/saveProfileSchema" target="_blank">server REST API</a> or follow <a href="http://kaaproject.github.io/kaa/docs/v0.10.0/Programming-guide/Key-platform-features/Endpoint-profiles/#setting-client-side-ep-profile-schema" target="_blank">these instructions</a>for Administration UI.
 	</li>
 </ol>
 
 <p>
-	The same data structure is used for the <b>server-side EP profile schema</b>.
+	The same data structure is used for the <b>endpoint server profile schema</b>.
```
Comment (hunch7) [inline profilingdemo/resources/projects.xml]: s/server-side profile schema of the endpoint

File: profilingdemo/resources/projects.xml [hunk 4]
```diff
@@ -729,7 +729,7 @@ The following configuration schema is used for this sample application.
 </pre>
 
 <p>
-	To view and edit this schema in the Administration UI, open the <b>Configuration</b> page under the <b>Schemas</b> section of your application and select the <b>PagerConfiguration</b> schema.
+	To view and edit this schema in the Administration UI, open the <b>Configuration</b> page under the <b>Schemas</b> section of your application and select the <b>Configuration schema</b>.
```
Comment (hunch7) [inline profilingdemo/resources/projects.xml]: s/configuration schema

**Review by hunch7 — CHANGES_REQUESTED** [review]
Please see my comments.

**Review by hunch7 — APPROVED** [review]

**Review by Slash32 — APPROVED** [review]

**Comment by Slash32** [comment]
@kaa-leeroy test this please

**Comment by Slash32** [comment]
@kaa-leeroy test this please

**Comment by Slash32** [comment]
@kaa-leeroy test this please

**Pull request merged**